### PR TITLE
Added browser tests after deploy

### PR DESCRIPTION
**Pull request by quarckster**
**What**:
QE use library https://github.com/quarckster/widgetastic.patternfly4 to run UI tests in Selenium. In order to keep up the changes in this repo it would be nice to trigger a travis job with browser tests that are described here https://github.com/quarckster/widgetastic.patternfly4/tree/master/testing. 

Solves #1315

### CODE REVIEW

**Comment by dgutride** [comment]
Wow - this is great, is there an open issue on github tracking this work so we can understand the impact (and how the PatternFly team can help with this effort)?  We won't be able to accept this in without at least an issue created on this repo that relates to this.  @quarckster

File: .travis/trigger_browser_tests.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+BODY='{"request": {"branch": "master"}}'
+
+curl -s -X POST \
+   -H "Content-Type: application/json" \
+   -H "Accept: application/json" \
+   -H "Travis-API-Version: 3" \
+   -H "Authorization: token $QUARCKSTER_TOKEN" \
```
Comment (dgutride) [inline .travis/trigger_browser_tests.sh]: Do you need us to set this token on travis?  I see it has been set, but it needs to be set by somebody on the PF team so we can keep track of what is used.  @karelhala @quarckster

Comment (quarckster) [reply .travis/trigger_browser_tests.sh]: Yes, please set up this token in Travis settings. I've already provided token to @karelhala.

**Comment by dgutride** [comment]
@quarckster - this needs to be updated from mastser and then we can take it in

**Comment by quarckster** [comment]
@zallen please review

**Comment by quarckster** [comment]
@redallen please review

File: .travis.yml [hunk 2]
```diff
@@ -91,3 +91,6 @@ jobs:
         - cp -r packages/patternfly-4/react-docs/public/assets .public/assets
         - cp -r .out .public/patternfly-3
         - yarn run surge --project .public --domain patternfly-react.surge.sh
+
+after_deploy:
+  - .circleci/trigger_browser_tests.sh
```
Comment (redallen) [inline .travis.yml]: This needs to be in the deploy step. Do:
 ```
    - stage: Deploy
      name: NPM and Github Release
      install: skip
      script:
        - .circleci/release.sh
        - .circleci/trigger_browser_tests.sh
```
and be sure to `chmod +x .circleci/trigger_browser_tests.sh`.

Comment (quarckster) [reply .travis.yml]: @redallen but it really needs to be performed after deploy. Does it make sense to create additional stage?

Comment (redallen) [reply .travis.yml]: Only if `.circleci/release.sh` returns 0 will our package be released to NPM for you to use, and your script will run. The `after_deploy` stage would also fire up another executor that would run for 1m 30s when we can just use that one after it releases.

Comment (quarckster) [reply .travis.yml]: @redallen I'm mostly interested in updating this page http://patternfly-react.surge.sh/patternfly-4/. Does `.circleci/release.sh` updates it as well?

Comment (redallen) [reply .travis.yml]: In that case, put it after `- yarn run surge --project .public --domain patternfly-react.surge.sh`.

Comment (quarckster) [reply .travis.yml]: @redallen thank you very much for the help

**Comment by quarckster** [comment]
@dgutride I rebased the PR and aligned it with @redallen comments.

**Pull request merged**